docs/resources/users_permission.md

```diff
@@ -0,0 +1,89 @@
+---
+page_title: "keycloak_users_permissions Resource"
+---
+
+# keycloak_users_permissions
+
+Allows you to manage all users Scope Based Permissions.
+
+This is part of a preview keycloak feature. You need to enable this feature to be able to use this resource.
+More information about enabling the preview feature can be found here: https://www.keycloak.org/docs/latest/server_installation/#profiles
```

Comment:
Which of the features listed here is actually needed?  I assume `admin_fine_grained_authz`?